tools/github-actions/audit/action.yml

```diff
@@ -10,11 +10,11 @@ inputs:
     required: false
     default: 'true'
   recursive:
-    description: 'Audit transitive dependencies as well'
+    description: 'Audit transitive dependencies as well (Yarn Berry only)'
     required: false
     default: 'true'
   environment:
-    description: 'Which environments to cover'
+    description: 'Which environments to cover ("production" or "development")'
```

Comment:
you could use `options` input in this case

Reply:
I'm not sure we can set `type: choice` and `options` for inputs of a GitHub Action, it seems to be for workflow inputs
correct me if I'm wrong

Reply:
Yup I think you're right